internal/cloud/backend_test.go

```diff
@@ -232,6 +233,36 @@ func TestCloud_configWithEnvVars(t *testing.T) {
 			},
 			expectedOrganization: "hashicorp",
 		},
+		"with no hostname specified": {
+			config: cty.ObjectVal(map[string]cty.Value{
+				"hostname":     cty.NullVal(cty.String),
+				"token":        cty.NullVal(cty.String),
+				"organization": cty.StringVal("hashicorp"),
+				"workspaces": cty.ObjectVal(map[string]cty.Value{
+					"name": cty.StringVal("prod"),
+					"tags": cty.NullVal(cty.Set(cty.String)),
+				}),
+			}),
+			vars: map[string]string{
+				"TF_HOSTNAME": "app.terraform.io",
+			},
+			expectedHostname: "app.terraform.io",
+		},
+		"with hostname and env var specified": {
+			config: cty.ObjectVal(map[string]cty.Value{
+				"hostname":     cty.StringVal("app.terraform.io"),
```

Comment:
GitHub should be able to access `app.staging.terraform.io`. Can you change to that hostname instead?